repos/system_upgrade/el7toel8/actors/irssi/actor.py

```diff
@@ -0,0 +1,30 @@
+from leapp.actors import Actor
+from leapp.models import InstalledRedHatSignedRPM
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+from leapp.reporting import Report
+from leapp.libraries.common.reporting import report_with_remediation
+
+
+class Irssi(Actor):
```

Comment:
Could we have a more descriptive name for this actor? Maybe something like "CheckIrssiInstalled"?

Reply:
I renamed all to checkirssi (including the filename).

Reply:
If you need different names, please tell me what to rename to what.